### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by Juanse7793**
In this PR I fulfilled the following requirements:

## Requirements :rocket: 
-  Create a method called `current_user` in your ApplicationController to make current user data available to all controllers.
    -  It will return the first user from the database. :heavy_check_mark: 
    -  Don't worry about proper authentication at this time. You will add it in the upcoming projects.
- Create forms to perform the following functions: :100: 
    - Creates a Post on behalf of the `current_user` (use the method that you created in your ApplicationController).
    - Create a comment on behalf of the `current_user` (use the method that you created in your ApplicationController).
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -6,7 +6,41 @@ def index
...
+    redirect_to user_post_path(current_user, @post)
+  end
+
+  def comment
+    @post = Post.find(params[:id])
+    p params
+    @comment = Comment.new(author: current_user, post: @post, text: params[:text])
+    @comment.save
+    redirect_to user_post_path(current_user, @post)
+  end
```
Comment (omar-labana) [inline app/controllers/posts_controller.rb]: - [x] Let's refactor this section a bit. Please move the like and comment logic to separate controllers(they will have their own CRUD logic in a later milestone) with the required actions. Also, please don't forget to change the routes config accordingly.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Juanse7793, 

Excellent work so far working on your first blog with rails! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (Juanse7793) [reply app/controllers/posts_controller.rb]: Done :100:

**Review by keroloussamy — APPROVED** [review]
Hi @Juanse7793, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**